samples/RhythmGame/src/main/cpp/Game.cpp

```diff
@@ -94,10 +94,9 @@ void Game::stop(){
 
     if (mAudioStream != nullptr){
         mAudioStream->close();
+        mAudioStream = nullptr;
```

Comment:
Add delete mAudioStream;

This doc probably needs updating!
https://github.com/google/oboe/blob/master/docs/FullGuide.md#closing-an-audio-stream

Reply:
There must be a better way than this. Maybe for Oboe v2.0 we should change `openStream` to return a smart pointer where `stream->close()` is called when the stream is destroyed.

Reply:
Anyway - done :)